homeassistant/components/guardian/strings.json

```diff
@@ -5,7 +5,7 @@
       "user": {
         "description": "Configure a local Elexa Guardian device.",
         "data": {
-          "ip_address": "IP Address",
+          "ip_address": "[%key:common::config_flow::data::ip%]",
```

Comment:
Please contain the changes to 1 integration, or 1 key

Reply:
My bad, didnt create a separate branch